README.md

```diff
@@ -93,6 +93,12 @@ The file can be found in the [extras](extras) folder.
 #### Compress/Convert your music files
 ([#11](https://github.com/nims11/IPod-Shuffle-4g/issues/11)) Shuffle is short on storage, and you might want to squeeze in more of your collection by sacrificing some bitrate off your files. In rarer cases, you might also possess music in formats not supported by your ipod. Although `ffmpeg` can handle almost all your needs, if you are looking for a friendly alternative, try [Soundconverter](http://soundconverter.org/).
 
+#### Podcast support
+Place podcast tracks in `iPod_Control/Podcasts` to generate playlists. These tracks will be skipped when shuffling, will be marked to remember their last playback position, and won't be included in the "All Songs" playlist.
+
+#### Use symlinks to create playlists
+Symlinks will be followed to their original location on the iPod Shuffle. This allows creating additional playlists without duplicating physical files or duplicate entries in the iTunesSD database.
```

Comment:
What do you mean by that? I dont understand the symlink part. Why would you want to place a podcast inside the podcast directory AND the music directory?

Also is there also support for autogenerating podcast playlists and voiceover names?

Reply:
In my case I have podcasts in a structure like this:
~~~
Podcasts/
    Playlist/
        1 feed 2 - episode 1 link
        2 feed 1 - episode 2 link
    Feed 1/
        episode 1
        episode 2
    Feed 2/
        episode 1
        episode 2
~~~

So if I want to listen to an ordered playlist that I've put together I can do that. Or I can listen to multiple episodes from the same feed.
You could make a playlist of songs and podcasts using this method, but I intended it for listening to feed tracks and playlist tracks.
Should I clarify that in the readme?

Reply:
I still dont understand. We have a feature that autogenerates playlists. Why is that not sufficient?

Reply:
That's a good point and something I hadn't thought about using in this case (something like `-i {album}` as long as all the podcasts have the show names in that location. Still though, following symlinks allows creating multiple playlists that reference the same files and database entries without creating duplicates (disk space and database bloat). This is probably a more common pattern for actual music.

~~~
Music/
    Artist 1/ # files on disk
        01 a1 s1
        02 a1 s2
        03 a1 s3
    Artist 2/ # files on disk
        01 a2 s1
        02 a2 s2
        03 a2 s3
    All Songs/ # generated
        a1 s1
        a1 s2
        a1 s3
        a2 s1
        a2 s2
        a2 s3
    Playlist 1/ # created with links
        01 a1 s1 link
        02 a2 s1 link
        03 a1 s2 link
        04 a2 s2 link
        05 a1 s3 link
        06 a2 s3 link
    Playlist 2/ # created with links
        01 a1 s3 link
        02 a2 s3 link
        03 a1 s2 link
        04 a2 s2 link
        05 a1 s1 link
        06 a2 s1 link
~~~

So you can have multiple Artist / Album folders and multiple playlists that reference those other tracks, without duplicating the files on the iPod. And because the links are followed to the original files when building the database, the database doesn't get bloated with duplicate tracks either. If someone has a project that uses the playback statistics this would keep those accurate instead of being split among different copies of the same songs.
So, the auto-generating stuff could indeed be better for my use case, but following symlinks does still have some utility as another way to handle playlists.

Reply:
In your case I'd create a dedicated m3u playlist and I would not frickel with symlinks. That does not make sense to me. It would also have all other benefits you mentioned.

Apart from the symlink discussion I dont see why this is related to podcast. I understand that a song might be in multiple playlists, but a podcast? I am not sure if your solution accepts m3u files for podcasts or the autogenerated playlists using the folders, but if it does not, you should fix that in my opinion.

So an easy solution is always better than something complex. I'd be happy if you can give it another cleanup round :-)

Reply:
Symlinks aren't related to podcasts really, they kinda just rode in. It's less complex for me to work with those and I think it's useful to keep the database clean, but if it's not desired I can pull that out. I haven't tested m3u or id3 playlists, but both should work fine. I'll give it a look and update the PR later today.